lib/scheduler.pm

```diff
@@ -70,6 +70,19 @@ sub get_test_suite_data {
     return $test_suite_data;
 }
 
+
+=head2 get_data_value_from
+
+Returns specific value of test_suite_data.
+
+=cut
+
+sub get_data_value_from {
```

Comment:
I believe you didn't want to return the same value that you passed and call the function instead.

Reply:
know that i am thinking this is not doing what was indented to. I will remove it. the idea was to make a small step to avoid to have duplicated test data. Anyway. ignore